groovy/inductor/inductor_xpu_backend_e2e.groovy

```diff
@@ -0,0 +1,304 @@
+node(env.nodes_label){
+    cleanWs()
+    stage('setup env') {
+        println('================================================================')
+        println('setup nodes env')
+        println('================================================================')
+        sh '''
+        set -e
+        set +x
+        if [ -e ${HOME}/miniconda3/etc/profile.d/conda.sh ];then
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            if [ -d ${HOME}/miniconda3/envs/${conda_env} ];then
+                conda activate ${conda_env}
+            else
+                conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+            fi
+        else
+            wget -q -e "https_proxy=http://proxy-us.intel.com:912" https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh -b
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+        fi
+
+        # set gpu governor
+        if [[ -z "${USER_PASS}" ]];then USER_PASS="gta";fi
+        for i in $(seq 0 $(($(nproc)-1)))
+        do
+            echo "${USER_PASS}" | sudo -S sh -c "echo performance > /sys/devices/system/cpu/cpu$i/cpufreq/scaling_governor"
+        done
+        '''
+    }//stage
+    stage('Install Dependency') {
+        println('================================================================')
+        println('Install Dependency')
+        println('================================================================')
+            sh '''
+            set -e
+            set +x
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            conda activate ${conda_env}
+            source ${HOME}/env.sh 
+            python -m pip install numpy --upgrade --no-cache --force-reinstall
+
+            if [ ! -d inductor-tools ];then
+                git clone --depth=1 https://github.com/chuanqi129/inductor-tools.git
+            fi
+
+            if [[ -n ${torch_whl} ]] && [[ -n ${ipex_whl} ]];then
```

Comment:
where the `torch_whl` `ipex_whl` come from? by Jenkins job parameters?

Reply:
They can set by Jenkins Job parameters: https://jenkins-aten-caffe2.sh.intel.com/job/IPEX-TorchInductor-XPU-Backend/

Reply:
Please show all Jenkins job parameters at the beginning of the groovy, to help check whether we passed excepted values.
like this https://github.com/chuanqi129/inductor-tools/blob/yudong/aws_auto/groovy/inductor_images.groovy#L1-L64